components/pedal-details-content/index.js

```diff
@@ -1,4 +1,5 @@
 import React, { useState } from "react";
+import { useRouter } from "next/router";
 import { pedals } from "../../lib/pedalfx-data";
 import PedalMainPage from "../pedal-details-main/index";
```

Comment:
As you are importing from the `index.js` file in this directory, you can omit the filename in your import statements, e.g.: 

```suggestion
import PedalMainPage from "../pedal-details-main";
```

Applies to all of your component imports.

Reply:
Thanks, I will tidy that.